kmod/src/format.h

```diff
@@ -86,11 +86,15 @@ struct scoutfs_timespec {
 	__u8 __pad[4];
 };
 
-/* XXX ipv6 */
+/*
+ * ipv4 address is stored in addr[0].
+ * ipv6 support (TBD) will use all 4 and set is_ipv6_addr.
+ */
 struct scoutfs_inet_addr {
-	__le32 addr;
+	__le32 addr[4];
 	__le16 port;
-	__u8 __pad[2];
+	__u8 is_ipv6_addr;
+	__u8 __pad[5];
 };
```

Comment:
By the principle of least surprise, let's make this look like the sockaddr union in the kernel.  It'd be 16 bytes instead of 24 and the family would differentiate between whatever types instead of having a u8 boolean.  I suspect we'll want to have the current code use the family to recognize ipv4 to make the eventual addition of ipv6 a lot cleaner, but it's a bit hazy.  Maybe we should have a chat about it.